tests/pipelines/test_pipelines.py

```diff
@@ -327,28 +327,28 @@ def test_download_no_openvino_by_default(self):
     def test_download_no_onnx_by_default(self):
         with tempfile.TemporaryDirectory() as tmpdirname:
             tmpdirname = DiffusionPipeline.download(
-                "hf-internal-testing/tiny-random-OnnxStableDiffusionPipeline",
+                "hf-internal-testing/tiny-stable-diffusion-pipe-indexes",
                 cache_dir=tmpdirname,
+                use_safetensors=False,
             )
 
             all_root_files = [t[-1] for t in os.walk(os.path.join(tmpdirname))]
             files = [item for sublist in all_root_files for item in sublist]
 
-            # make sure that by default no onnx weights are downloaded
+            # make sure that by default no onnx weights are downloaded for not ONNX pipelines
             assert all((f.endswith(".json") or f.endswith(".bin") or f.endswith(".txt")) for f in files)
             assert not any((f.endswith(".onnx") or f.endswith(".pb")) for f in files)
 
         with tempfile.TemporaryDirectory() as tmpdirname:
             tmpdirname = DiffusionPipeline.download(
                 "hf-internal-testing/tiny-random-OnnxStableDiffusionPipeline",
                 cache_dir=tmpdirname,
-                use_onnx=True,
             )
 
             all_root_files = [t[-1] for t in os.walk(os.path.join(tmpdirname))]
             files = [item for sublist in all_root_files for item in sublist]
 
-            # if `use_onnx` is specified make sure weights are downloaded
+            # make sure that by default onnx weights are downloaded for ONNX pipelines
```

Comment:
I think it might be better to move this into it's own test with a `require_onnxruntime` decorator.